ptcg_cockatrice.xml

```diff
@@ -1336,7 +1336,7 @@ Whenever Exeggutor blocks a Pokémon with flying, put two -1/-1 counters on that
 			<type>Pokémon - Ground</type>
 			<pt>3/2</pt>
 			<tablerow>2</tablerow>
-			<text>Prowess</text>
+			<text>Vigilance</text>
```

Comment:
nice change, works well with marowak

Reply:
That was the idea. Also better fitting its colors. Green never gets prowess and it's tertiary for white, hasn't happened since Khans.